.eslintrc

```diff
@@ -19,6 +19,7 @@
         "ExportDeclaration": { "minProperties": 10, "multiline": true, "consistent": true }
       }
     ],
+    "no-use-before-define": "off",
```

Comment:
Already covered by [`typescript-eslint/no-use-before-define`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-use-before-define.md).
The JS one is reporting some weird errors.